### ISSUE TITLE
CommentsIndentation Check false-positive when a singleline comment follows a block comment

### ISSUE THREAD

**Issue by Vladlis** [checkstyle/checkstyle#3126]
```
vlad@vlad-N61PC-M2S ~/cli $ java -jar checkstyle-6.17-all.jar com.puppycrawl.tools.checkstyle.Main -c my_check.xml Test.java
Starting audit...
[ERROR] /home/vlad/cli/Test.java:6: Comment has incorrect indentation level 8, expected is 16, indentation should be the same level as line 5. [CommentsIndentation]
Audit done.
Checkstyle ends with 1 errors.
vlad@vlad-N61PC-M2S ~/cli $ cat Test.java
public class Test {
    void foo1() {
        foo1();
        /*
         line 5 */
        // violation - line 6
    }
}
vlad@vlad-N61PC-M2S ~/cli $ cat my_check.xml
<?xml version="1.0"?>
<!DOCTYPE module PUBLIC
          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
          "http://www.puppycrawl.com/dtds/configuration_1_3.dtd">


<module name = "Checker">
    <module name="TreeWalker">
         <module name="CommentsIndentation">
         </module>
    </module>
</module>
```
Expected - no violations.
